src/ui/components/stepCard/StepCard.native.test.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+import { render, screen } from 'tests/utils'
+import { BicolorAroundMe } from 'ui/svg/icons/BicolorAroundMe'
+
+import { StepCard } from './StepCard'
+
+describe('<StepCard />', () => {
+  it('should render title and icon', () => {
+    const title = 'This is a StepCard'
+    const icon = <MockIcon />
+    render(<StepCard title={title} icon={icon} />)
+
+    expect(screen.getByText(title)).toBeTruthy()
+    expect(screen.getByTestId('stepcard-icon')).toBeTruthy()
+  })
+
+  it('should render subtitle', () => {
+    const title = 'This is a StepCard'
+    const subtitle = 'This is a subtitle'
+    const icon = <MockIcon />
+    render(<StepCard title={title} subtitle={subtitle} icon={icon} />)
+
+    expect(screen.getByText(subtitle)).toBeTruthy()
+  })
+
+  it('should disable the wrapper when disabled is true', () => {
+    const title = 'This is a StepCard'
+    const icon = <MockIcon />
+    render(<StepCard title={title} icon={icon} disabled />)
+
+    expect(screen.getByTestId('stepcard-container')).toHaveStyle({ borderColor: '#CBCDD2' })
```

Comment:
Il te manque un autre test dans ce cas : il faut tester que ton title est gris également.
Par ailleurs plutôt que de mettre un `borderColor` en dur tu peux aussi faire
```suggestion
    expect(screen.getByTestId('stepcard-container')).toHaveStyle({ borderColor: theme.colors.TACOULEUR })
```